.github/workflows/build.yml

```diff
@@ -27,8 +27,14 @@ jobs:
       run: dotnet restore
     - name: Build
       run: dotnet build --configuration Release --no-restore
+    - name: Create testing environemnt stack
+      run: docker-compose -f test/docker/docker-compose.yml up -d
+    - name: Check running containers
```

Comment:
I think we need to wait on mysql to start. I'll update the PR with the needed bootstrap script.